source/common/http/conn_manager_impl.h

```diff
@@ -290,16 +327,32 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     ActiveStream(ConnectionManagerImpl& connection_manager);
     ~ActiveStream();
 
+    // Indicates which filter to start the iteration with.
+    enum class FilterIterationStartState { Always_start_from_next, Can_start_from_current };
```

Comment:
nit: `AlwaysStartFromNext` and `CanStartFromCurrent`

Reply:
Fixed. Thanks!